sklearn/datasets/lfw.py

```diff
@@ -450,17 +451,19 @@ def fetch_lfw_pairs(subset='train', data_home=None, funneled=True, resize=0.5,
     -------
     The data is returned as a Bunch object with the following attributes:
 
-    data : numpy array of shape (2200, 5828)
+    data : numpy array of shape (2200, 5828). NOTE: Shape depends on
+           ``subset``.
         Each row corresponds to 2 ravel'd face images of original size 62 x 47
-        pixels. Changing the ``slice_`` or resize parameters will change the shape
-        of the output.
+        pixels. Changing the ``slice_`` or resize parameters will change the
```

Comment:
Remove the NOTE: and just add it to the docs

``slice_``, ``resize`` or ``subset`` parameters.